Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -696,6 +696,21 @@ AuthorAndUserStylesEnabled:
     WebCore:
       default: true
 
+AutomaticLiveCaptioningEnabled:
+  type: bool
+  status: unstable
+  category: media
+  humanReadableName: "Live Caption Generation for Video"
+  humanReadableDescription: "Enables live, automatically-generated captions for video"
+  condition: ENABLE(VIDEO)
```

Comment:
I know you're planning on adding a compile-time flag wrapping the feature. This would probably be the right place to put it, so that this setting doesn't show up for all ports which support VIDEO.